backtrace-library/src/androidTest/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbsTest.java

```diff
@@ -327,10 +330,10 @@ public void testQueueFileRollover() {
                 assertNotNull(parsedBreadcrumb.getJSONObject("attributes").get("From Thread"));
                 assertEquals("manual", parsedBreadcrumb.get("type"));
                 assertEquals("info", parsedBreadcrumb.get("level"));
-                // Timestamp should be convertible to an int
+                // Timestamp should be convertible to a long
                 assertTrue(parsedBreadcrumb.get("timestamp") instanceof Long);
-                // Id should be convertible to an int
-                assertTrue(((int)parsedBreadcrumb.get("id")) > 450);
+                assertTrue(((int) parsedBreadcrumb.get("id")) > 450);
+                assertTrue((int) parsedBreadcrumb.get("id") <= 1000);
```

Comment:
you can use `numIterations`  instead of 1000

Reply:
Good point, will do!